pkg/agent/const.go

```diff
@@ -41,39 +41,34 @@ const (
 
 // cloud-init (i.e. ARM customData) source file references.
 const (
-	kubernetesNodeCustomDataYaml      = "linux/cloud-init/nodecustomdata.yml"
-	kubernetesCSECommandString        = "linux/cloud-init/artifacts/cse_cmd.sh"
-	kubernetesCSEStartScript          = "linux/cloud-init/artifacts/cse_start.sh"
-	kubernetesCSEMainScript           = "linux/cloud-init/artifacts/cse_main.sh"
-	kubernetesCSEHelpersScript        = "linux/cloud-init/artifacts/cse_helpers.sh"
-	kubernetesCSEHelpersScriptUbuntu  = "linux/cloud-init/artifacts/ubuntu/cse_helpers_ubuntu.sh"
-	kubernetesCSEHelpersScriptMariner = "linux/cloud-init/artifacts/mariner/cse_helpers_mariner.sh"
-	kubernetesCSEInstall              = "linux/cloud-init/artifacts/cse_install.sh"
-	kubernetesCSEInstallUbuntu        = "linux/cloud-init/artifacts/ubuntu/cse_install_ubuntu.sh"
-	kubernetesCSEInstallMariner       = "linux/cloud-init/artifacts/mariner/cse_install_mariner.sh"
-	kubernetesCSEConfig               = "linux/cloud-init/artifacts/cse_config.sh"
-	kubernetesCSESendLogs             = "linux/cloud-init/artifacts/cse_send_logs.py"
-	kubernetesCSERedactCloudConfig    = "linux/cloud-init/artifacts/cse_redact_cloud_config.py"
-	kubernetesCISScript               = "linux/cloud-init/artifacts/cis.sh"
-	kubernetesHealthMonitorScript     = "linux/cloud-init/artifacts/health-monitor.sh"
-	// kubernetesKubeletMonitorSystemdTimer= "linux/cloud-init/artifacts/kubelet-monitor.timer" // TODO enable,
-	kubernetesKubeletMonitorSystemdService    = "linux/cloud-init/artifacts/kubelet-monitor.service"
-	kubernetesDockerMonitorSystemdTimer       = "linux/cloud-init/artifacts/docker-monitor.timer"
-	kubernetesDockerMonitorSystemdService     = "linux/cloud-init/artifacts/docker-monitor.service"
```

Comment:
Lines 58, 59, 60, 61, 62 were moved. Kinda hard to tell because it caused a reformat of the other variables.